config.tfvars

```diff
@@ -55,6 +55,7 @@ jira_replica_count = 1
 
 # By default, Jira Software will use the version defined in the Helm chart. If you wish to override the version, uncomment
 # the following line and set the jira_version_tag to any of the versions available on https://hub.docker.com/r/atlassian/jira-software/tags
+#jira_image_repository = "atlassian/jira-software"
```

Comment:
Maybe this location is not the best to land this line. The comment on lines 57 & 58 refering to jira version tag as the `following line` which now is different variable. 

Can we move it to another line and do you think if any comment is needed for `jira_image_repository` in config file?